client/context/EditorContext.tsx

```diff
@@ -1,8 +1,22 @@
 import React, { createContext, useContext, useReducer } from "react";
-import { editObjectCatalogUrl, getObjectDetailsUrl } from "../util/routes";
+import { editObjectCatalogUrl, getObjectChildrenUrl, getObjectDetailsUrl } from "../util/routes";
 import { useFetchContext } from "./FetchContext";
 import { extractFirstMetadataValue as utilExtractFirstMetadataValue } from "../util/metadata";
 
+export interface ObjectDetails {
```

Comment:
We should probably rename the Children type. The reason for two separate interfaces is that these represent responses from different API endpoints. ObjectDetails describes a single object. The Children interface actually describes a single page from a paginated Solr response, which includes the count of the total result set, the offset of the page, and a docs array containing a page of documents from the overall set. Maybe if we rename this to ChildrenResultPage it would make more sense. What do you think?

Reply:
I've renamed the interface as proposed above, but happy to rename it again if you have a better suggestion.